mne/beamformer/tests/test_lcmv.py

```diff
@@ -688,6 +689,61 @@ def test_lcmv_reg_proj(proj):
                         weight_norm='nai', rank=None, verbose=True)
     want_rank = 302  # 305 good channels - 3 MEG projs
     assert filters['rank'] == want_rank
+    # And also with and without noise_cov
+    with pytest.raises(ValueError, match='several sensor types'):
+        make_lcmv(epochs.info, forward, data_cov, reg=0.05,
+                  noise_cov=None)
+    epochs.pick_types('grad')
+    kwargs = dict(reg=0.05, pick_ori=None, weight_norm=weight_norm)
+    filters_cov = make_lcmv(epochs.info, forward, data_cov,
+                            noise_cov=noise_cov, **kwargs)
+    filters_nocov = make_lcmv(epochs.info, forward, data_cov,
+                              noise_cov=None, **kwargs)
+    ad_hoc = mne.make_ad_hoc_cov(epochs.info)
+    filters_adhoc = make_lcmv(epochs.info, forward, data_cov,
+                              noise_cov=ad_hoc, **kwargs)
+    evoked = epochs.average()
+    stc_cov = apply_lcmv(evoked, filters_cov)
+    stc_nocov = apply_lcmv(evoked, filters_nocov)
+    stc_adhoc = apply_lcmv(evoked, filters_adhoc)
+
+    # Compare adhoc and nocov: scale difference is necessitated by using std=1.
```

Comment:
@britta-wstnr let's figure it out later. For now I've added tests that the amplitudes are as expected for None, NAI and unit-noise-gain (the first two not depending on `noise_cov`, the last being normalized or not depending on whether or not `noise_cov is None`). The amplitudes you get now should be like the ones you get in `master`, can you check?

If we think it makes sense, we can remove the `std=1.` from the `_check_one_ch_type` function. I think it would since it makes things more consistent (as shown now by the tests, I think), but let's do it later so this PR isn't held up.